packages/shared/modules/programs-table/components/programs-table/program-table-row-short.tsx

```diff
@@ -133,7 +133,7 @@ const ProgramTableRowShort: React.FC<
         {periodStarts && (
           <ProgramPeriodPie
             condition={status !== STATUS.CLOSED}
-            loader={<>t("program-period.program-closed"))</>}
+            loader={<>{t("program-period.program-closed")}</>}
```

Comment:
Using a text will be enough
```suggestion
            loader={t("program-period.program-closed")}
```